src/config.rs

```diff
@@ -49,5 +49,34 @@ pub fn get_backups_flder() -> PathBuf {
 
 #[cfg(target_family = "unix")]
 pub fn get_boilr_links_path() -> PathBuf {
-    get_config_folder().join("links").to_path_buf()
+    get_config_folder().join("links")
+}
+
+
+#[cfg(test)]
+mod tests{
+    use std::path::PathBuf;
+
+    use super::get_config_folder;
+
+    #[test]
+
+    #[cfg(target_family = "unix")]
+    fn check_return_config_format(){
+        // Parameters for set environment 'XDG_CONFIG_HOME'
+        // std::env::set_var("XDG_CONFIG_HOME", std::env::var("HOME").unwrap()+"/.config/xdg");
```

Comment:
Maybe turn this into two tests, instead of having some commented out code?

Reply:
Good idea. I will update later.

Reply:
```
 *  Executing task: cargo test --package boilr --bin boilr -- config::tests --nocapture 

   Compiling boilr v1.5.1 (/workspaces/BoilR)
    Finished test [unoptimized + debuginfo] target(s) in 33.18s
     Running unittests src/main.rs (target/debug/deps/boilr-23166f7cd679110a)

running 2 tests
test config::tests::check_return_xdg_config_path ... ok
test config::tests::check_return_config_path ... ok

test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 24 filtered out; finished in 0.00s
```